SDK/simVis/View.cpp

```diff
@@ -1556,10 +1562,11 @@ void View::enableWatchMode(osg::Node* watched, osg::Node* watcher)
         simVis::EarthManipulator* manip = dynamic_cast<simVis::EarthManipulator*>(getCameraManipulator());
         if (manip && manip->isTethering())
         {
+          osg::ref_ptr<osg::Node> tetherNode = manip->getViewpoint().getNode();
```

Comment:
No matching function call to osgEarth::Viewpoint::getNode()
Candidate expects 1 argument, 0 provided

Reply:
added ifdef